distributed/dashboard/utils.py

```diff
@@ -1,5 +1,7 @@
+from distutils.version import LooseVersion
```

Comment:
We recently switched to using `packaging` for version parsing instead of `distutils`, which is now deprecated (xref https://github.com/dask/distributed/pull/5624). Could you switch to using `packaging` here instead

Reply:
Oh, I guess it considers `<Version('3.0.0.dev3')>` to be "less than" 3.0? That's really unfortunate for testing/integration purposes. What's the right approach with packaging here?

Reply:
OK hopefully third time is the charm 748aee3

Reply:
Ah, that is confusing. The `.major` approach seems reasonable